### PR TITLE
fix(Breadcrumbs): css and api alignment

### PR DESCRIPTION
**Pull request by eirikbacker**
Part of https://github.com/digdir/designsystemet/issues/2295 and https://github.com/digdir/designsystemet/issues/2221
Moves back link into `nav` as this make sense also from a11y perspective after discussion with NRK

### CODE REVIEW

File: packages/react/src/components/Breadcrumbs/Breadcrumbs.tsx [hunk 1]
```diff
@@ -1,23 +1,26 @@
...
+  size?: 'sm' | 'md' | 'lg';
 } & HTMLAttributes<HTMLElement>;
 
-export const BreadcrumbsNav = forwardRef<HTMLElement, BreadcrumbsNavProps>(
+export const Breadcrumbs = forwardRef<HTMLElement, BreadcrumbsProps>(
   ({ 'aria-label': ariaLabel = 'Du er her:', className, ...rest }, ref) => (
     <nav
       aria-label={ariaLabel}
+      className={cl('ds-breadcrumbs', className)}
       ref={ref}
```
Comment (Barsnes) [inline packages/react/src/components/Breadcrumbs/Breadcrumbs.tsx:22]:
```suggestion
  ({ 'aria-label': ariaLabel = 'Du er her:', size = 'md', className, ...rest }, ref) => (
    <nav
      aria-label={ariaLabel}
      className={cl('ds-breadcrumbs', className)}
      data-size={size}
      ref={ref}
```
Changing size does currently not work, looks like you missed adding it?

**Review by Barsnes — DISMISSED** [review]

File: packages/react/src/components/Breadcrumbs/Breadcrumbs.tsx (hunk 1 above)
Comment (eirikbacker) [reply packages/react/src/components/Breadcrumbs/Breadcrumbs.tsx:22]: Oh, well spotted! Will fix!

Comment (eirikbacker) [reply packages/react/src/components/Breadcrumbs/Breadcrumbs.tsx]: Resolved ✅

File: .changeset/swift-forks-drop.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+"@digdir/designsystemet-css": patch
+"@digdir/designsystemet-react": patch
+---
+
+Breadcrumbs: Rename Breadcrumbs.Root to Breadcrumbs and remove Breadcrumbs.Nav
```
Comment (Barsnes) [inline .changeset/swift-forks-drop.md:6]:
```suggestion
Breadcrumbs: Rename `Breadcrumbs.Root` to `Breadcrumbs` and remove `Breadcrumbs.Nav`
```

**Review by Barsnes — APPROVED** [review]

**Pull request merged**